babel.config.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
           '@babel/preset-env',
           {
             targets: {
-              node: '14',
+              node: 'current',
```

Comment:
Shouldn't we set this to what we pin Node.js to in Nix?
https://github.com/status-im/status-mobile/blob/f5f1e6302d81a90faa6d56c2aa57ed32fdfd6411/nix/overlay.nix#L54-L55

Reply:
Makes sense.